sources/views/contacts.js

```diff
@@ -62,4 +72,10 @@ export default class Contacts extends JetView {
 		this.list.unselectAll();
 		this.show("/top/contacts/contactsForm");
 	}
+
+	contactsFilter() {
+		const value = this.$$("contactsFilterInput").getValue().toLowerCase();
+		this.list.filter(contact => contact.value.toLowerCase().indexOf(value) !== -1 ||
+			contact.Job.toLowerCase().indexOf(value) !== -1);
```

Comment:
Required features:
filtering contacts by all data attributes from the one input

I understood that contacts should be filtered not only by these two attributes but by all of them.

Reply:
Added all fields from contacts details to filter function.